server/app/services/program/StatusDefinitions.java

```diff
@@ -0,0 +1,71 @@
+package services.program;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.google.auto.value.AutoValue;
+import com.google.common.collect.ImmutableList;
+import java.util.Optional;
+import services.LocalizedStrings;
+
+public class StatusDefinitions {
```

Comment:
Consider marking this class as final to prevent subclassing (or at least make the decision to allow subclassing explicit in the future when needed).

Reply:
Making it final gives a warning about needing toString. Not exactly sure what that's about, but not worth the time right now.  We don't have a pattern of making these final so may want to do that enmasse